app/src/main/java/com/codesoom/assignment/Toy/application/ToyService.java

```diff
@@ -0,0 +1,96 @@
+package com.codesoom.assignment.Toy.application;
+
+
+import com.codesoom.assignment.Task.TaskNotFoundException;
+import com.codesoom.assignment.Toy.ToyNotFoundException;
+import com.codesoom.assignment.Toy.domain.Toy;
+import com.codesoom.assignment.Toy.domain.ToyJpaRepository;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+/**
+ * Toy Entity의 REST API
+ * GET - getAllToy(), getToyById()
+ * POST - createToy()
+ * UPDATE - updateToy()
+ * DELETE - deleteToy()
+ */
+@Service
+@Transactional
+public class ToyService {
+
+    private final ToyJpaRepository toyJpaRepository;
+
+    /**
+     * 생성자에 ToyJpaRepository 주입
+     * @param toyJpaRepository CrudRepository 상속받은 인터페이스
+     */
+    public ToyService(ToyJpaRepository toyJpaRepository) {
+        this.toyJpaRepository = toyJpaRepository;
+    }
+
+    /**
+     * 모든 Toy 목록 가져옴
+     * @return 모든 Toy 를 담고있는 list 반환
+     */
```

Comment:
메소드에 달아주는 JavaDoc은 리턴값을 위주로 설명해 주세요.
```suggestion
    /**
     * 모든 Toy 목록을 가져와 리턴합니다.
     * @return 모든 Toy 를 담고 있는 list
     */
```

Reply:
수정하였습니다

Reply:
제대로된 javadoc 작성법을 배웠습니다 감사합니다